### PR TITLE
ft(test): add eve support for tests

### PR DESCRIPTION
**Pull request by ThibaultRiviere**

### CODE REVIEW

File: eve/workers/master/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    && cat /tmp/bucketclient_packages.list | xargs apt-get install -qy \
+    && pip install pip==9.0.1 \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f /tmp/*_packages.list
+
+#
+# Install usefull nodejs dependencies
+#
+
+RUN npm install mocha -g
```
Comment (tcarmet) [inline eve/workers/master/Dockerfile]: Do you need mocha?

File: eve/workers/master/bucketclient_packages.list [hunk 2]
```diff
@@ -0,0 +1 @@
+nodejs
```
Comment (tcarmet) [inline eve/workers/master/bucketclient_packages.list]: If we only have one package for bucketclient I recommend putting `nodejs` directly into the dockerfile. What do you think?

File: eve/workers/master/Dockerfile (hunk 1 above)
Comment (ThibaultRiviere) [reply eve/workers/master/Dockerfile]: mocha is used to run the tests :)

File: eve/workers/master/bucketclient_packages.list (hunk 2 above)
Comment (ThibaultRiviere) [reply eve/workers/master/bucketclient_packages.list]: No prob for me it was just to keep the same format everywhere

Comment (ThibaultRiviere) [reply eve/workers/master/bucketclient_packages.list]: I think it's not a bad idea to try to stay consistent across ours projects

File: eve/workers/master/Dockerfile (hunk 1 above)
Comment (tcarmet) [reply eve/workers/master/Dockerfile]: ok, perfect

File: eve/workers/master/bucketclient_packages.list (hunk 2 above)
Comment (tcarmet) [reply eve/workers/master/bucketclient_packages.list]: It's a valid argument. Let's keep it that way. I think I did the same with S3

**Comment by ThibaultRiviere** [comment]
Merging without e2e since it doesn't touch the code base .

**Pull request merged**